resources/assets/react/screens/Asso/index.js

```diff
@@ -494,6 +510,7 @@ class AssoScreen extends React.Component {
 			};
 		}
 
+		this.user.isCotisant = user.types.contributorBde === true;
```

Comment:
À absolument éviter. 

`this.user` réfère aux données membre de l'association, la tu fais de l'injection d'une donnée qui n'a aucun impact sur un membre